sources/packages/backend/libs/test-utils/src/factories/designation-agreement-locations.ts

```diff
@@ -16,6 +22,7 @@ export async function saveFakeDesignationAgreementLocation(
   db: E2EDataSources,
   options: {
```

Comment:
Can we have the options as optional making a default numberofLocations as one, if not given. That will make the method easier for creating fakeDesignationAgreement for a single location.